### PR TITLE
Some initial examples

### PR DESCRIPTION
**Pull request by jfriel-oqc**
Just starting some examples, I'll continue to push more as I think of them, please push straight to branch with more.

Will add more comments into the files shortly, too. Just wanted to kick off the PR

### CODE REVIEW

File: examples/plot_pulse_buffers.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+np.set_printoptions(
+    edgeitems=3000000, linewidth=100000000, formatter=dict(float=lambda x: "%.3g" % x)
+)
+
+import logging
+
+logging.disable()
+
+
+def plot_physical_buffers(
```
Comment (chemix-lunacy) [inline examples/plot_pulse_buffers.py:21]: If you want to show this off build a throwaway engine and use the normal pipeline. While these are the seperate components to do it, the code itself is very much not what you should do and putting it as an example gives the wrong idea.

File: examples/plot_pulse_buffers.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from qat.purr.compiler.runtime import get_builder
+from qat.purr.integrations.qasm import Qasm3Parser
+
+np.set_printoptions(
+    edgeitems=3000000, linewidth=100000000, formatter=dict(float=lambda x: "%.3g" % x)
+)
+
+import logging
+
+logging.disable()
```
Comment (chemix-lunacy) [inline examples/plot_pulse_buffers.py:18]: What's with this?

File: examples/real_time_chip_sim.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+hw = get_default_RTCS_hardware()
+
+control_q = hw.get_qubit(0)
+target_q = hw.get_qubit(1)
+cr_channel = control_q.get_cross_resonance_channel(target_q)
+sync_channels = [cr_channel, control_q.get_drive_channel(), target_q.get_drive_channel()]
+prep = np.linspace(0.0, np.pi, 2)
+
+engine = RealtimeChipSimEngine(hw, auto_plot=True)
+runtime = QuantumRuntime(engine)
```
Comment (chemix-lunacy) [inline examples/real_time_chip_sim.py:22]: `get_runtime(...)` and just assign auto-plot via the model or after you've initialized it.

File: examples/builder.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+cr_channel = control_q.get_cross_resonance_channel(target_q)
+sync_channels = [
+    cr_channel,
+    control_q.get_drive_channel(),
+    target_q.get_drive_channel(),
+]
+prep = np.linspace(0.0, np.pi, 2)
+
+# sweep over a time range
+builder = (
```
Comment (chemix-lunacy) [inline examples/builder.py:17]: Just disable ypaf on any use of the builders or don't use the formatter.

**Comment by chemix-lunacy** [comment]
I might just make the changes directly if we're doing this as a group thing.

**Comment by keriksson-rosenqvist** [comment]
You'll need to rebase on the latest main version for the workflow to pass.

File: examples/plot_pulse_buffers.py (hunk 2 above)
Comment (jfriel-oqc) [reply examples/plot_pulse_buffers.py:18]: oh yeah can remove, I just don't like most the logging when I'm testing, it's noisy

File: examples/plot_pulse_buffers.py (hunk 1 above)
Comment (jfriel-oqc) [reply examples/plot_pulse_buffers.py:77]: yeah fair, I understand why to build this. The thinking was more behind breaking down the various buffer building steps. But think it's clear enough from here tbf, might add a comment to clarify what each step is

**Review by chemix-lunacy — APPROVED** [review]
Approving, any examples are better than none. May need to re-base due to what Kajsa mentioned.

**Review by hamidelmaazouz — APPROVED** [review]

**Comment by jfriel-oqc** [comment]
I have read the Contributor License Agreement and I hereby accept the Terms.

**Comment by jfriel-oqc** [comment]
I have read the Contributor License Agreement and I hereby accept the Terms.

**Comment by chemix-lunacy** [comment]
I have read the Contributor License Agreement and I hereby accept the Terms.

**Pull request merged**